src/Components/Search/Previews/Grids/MerchandisableArtworks.tsx

```diff
@@ -85,16 +91,23 @@ class MerchandisableArtworksPreview extends React.Component<
 export const MerchandisableArtworksPreviewFragmentContainer = createFragmentContainer(
   (props: MerchandisableArtworksPreviewProps) => {
     return (
-      <Subscribe to={[SearchBarState]}>
-        {(searchState: SearchBarState) => {
+      <Responsive>
+        {({ xs, sm, md }) => {
           return (
-            <MerchandisableArtworksPreview
-              searchState={searchState}
-              {...props}
-            />
+            <Subscribe to={[SearchBarState]}>
```

Comment:
Nested render props are scary, but I know we'll be cleaning up `<Subscribe to` with hooks soon. (No action necessary, just a comment.)